src/common/Marker/Marker.module.css

```diff
@@ -0,0 +1,5 @@
+.marker {
+  pointer-events: none;
+  stroke-dasharray: 4, 4;
```

Comment:
We probably want to make this solid by default but let the user do this by passing their own CSS classname to the marker.